src/spatialdata/_core/spatialdata.py

```diff
@@ -682,292 +585,6 @@ def transform_to_coordinate_system(
             elements[element_type][element_name] = transformed
         return SpatialData(**elements, table=sdata.table)
 
-    def add_image(
```

Comment:
Let's keep in mind that this PR removed the commented code here, since it will probably be useful for making the "lazy copy" approach, or similar, work in the future.